backend/server.js

```diff
@@ -20,10 +25,26 @@ mongoose
   .then(() => console.log('connected to DB'))
   .catch(console.error);
 
-app.get('/', (req, res) => {});
+app.get('/', (req, res) => {
+    res.sendFile('../frontend/index.html', function (err) {
+      if (err) {
+        next(err);
+      } else {
+        console.log('Sent:', 'index.html');
+      }
+    });
+});
 
+//loginRoute
 app.use('/login', loginRouter);
 
+//signUpRoute
+app.use('/signup', signUpRouter);
+
+//cardsRoute
+app.use('/cards', cardsRouter)
+
+
```

Comment:
For the proxy in webpack to work these routes would be to be `/api/login` or `/api/signup` or `/api/cards`